consensus/telemetry/metrics.go

```diff
@@ -0,0 +1,30 @@
+package consensus_telemetry
+
+const (
+	// How to define metrics in this file?
```

Comment:
Can this be moved into a shared telemetry README (since it'll be the same for every module) and reference that here?

Please apply this to any other places too.